File: chatwoot/csharp/src/OSEG.ChatwootExamples/DeleteAutomationRuleFromAccountExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class DeleteAutomationRuleFromAccountExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("api_access_token", "USER_API_KEY");

        try
        {
            new AutomationRuleApi(config).DeleteAutomationRuleFromAccount(
                accountId: 0,
                id: 0
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling AutomationRuleApi#DeleteAutomationRuleFromAccount: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-axios/src/DeleteAutomationRuleFromAccountExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "USER_API_KEY",
});

new api.AutomationRuleApi(configuration).deleteAutomationRuleFromAccount(
  0, // accountId
  0, // id
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling AutomationRuleApi#deleteAutomationRuleFromAccount:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
